pmdk.spec

```
@@ -1,41 +1,27 @@
 
-# rpmbuild options:
-#   --with ndctl
-#   --define _testconfig <path to custom testconfig.sh>
-#   --define _skip_check 1
-
 # do not terminate build if files in the $RPM_BUILD_ROOT
 # directory are not found in the %%files (without rpmem case)
 #define _unpackaged_files_terminate_build 0
 
 # Ignore an issue with unpackaged libpmem2 files
 %define _unpackaged_files_terminate_build 0
 
-# disable 'make check' on suse
 %if %{defined suse_version}
-    %define _skip_check 1
     %define dist .suse%{suse_version}
 %endif
 
 %global major 2
 %global minor 1
 %global bugrelease 0
 #%%global prerelease rc1
-%global buildrelease 1
+%global buildrelease 2
 
 %global _hardened_build 1
 
-# by default build without ndctl, unless explicitly enabled
-%bcond_with ndctl
-
 %define min_ndctl_ver 63
 %define _make_common_args EXTRA_CFLAGS="-Wno-error" NORPATH=1 BUILD_EXAMPLES=n BUILD_BENCHMARKS=n
 
-%if %{with ndctl}
-    %define make_common_args %{_make_common_args}
-%else
-    %define make_common_args %{_make_common_args} NDCTL_ENABLE=n PMEMOBJ_IGNORE_DIRTY_SHUTDOWN=y PMEMOBJ_IGNORE_BAD_BLOCKS=y
-%endif
+%define make_common_args %{_make_common_args}
```

If there is no use-case for `--without-ndctl` then this indirection through `%{_make_common_args}` is no longer necessary and just adds confusion (this suggestion may look goofy because GH doesn't really support suggesting on non-changed lines, but I suspect you will get the meaning regardless):

```suggestion
%define make_common_args EXTRA_CFLAGS="-Wno-error" NORPATH=1 BUILD_EXAMPLES=n BUILD_BENCHMARKS=n
```

---

Done